projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -59,16 +59,17 @@
 				} );
 			}
 
-			const jitmDismissButton = adminMenu.querySelector( '.dismissible-card__close-icon' );
-			if ( jitmDismissButton ) {
-				jitmDismissButton.addEventListener( 'click', function ( event ) {
+			adminMenu.addEventListener( 'click', function ( event ) {
+				if ( event.target.classList.contains( 'dismissible-card__close-icon' ) ) {
```

Comment:
I think this is not capturing correctly clicks on the close icon. I tried dismissing an upsell nudge on a Simple site with these changes applied, and I got redirected to the upsell nudge link:

https://user-images.githubusercontent.com/1233880/167865813-42ea7e06-f38e-42c5-8864-5bc8ec9985a1.mov

Reply:
Bummer! You must have been precise enough to click on the X, which isn't an SVG but is rather a `path` element. Fixed in the last commit. Mind giving it another spin?

Reply:
Maybe we can move this logic to the callback that renders the upsell? That way we can bind the click listener to the `.dismissible-card__close-icon` element as before, since it'd be already rendered.